flowise-embed/src/utils/jsonUtils.ts

```diff
@@ -39,3 +39,53 @@ function shouldReplaceWithNull(keyValue: any): boolean {
 
   return ['n/a', 'null', 'undefined', ''].includes(keyValue.toString().toLowerCase());
 }
+
+export function sanitizeToFlatArray(input: any): any[] {
+  const result: any[] = [];
+
+  function recursiveSanitize(value: any): void {
+    if (Array.isArray(value)) {
+      value.forEach(recursiveSanitize);
+    } else if (value && typeof value === 'object') {
+      Object.values(value).forEach(recursiveSanitize);
+    } else if (typeof value === 'string') {
+      const sanitized = value.replace(/\D/g, '');
+      if (sanitized) {
+        result.push(sanitized);
+      }
+    } else if (typeof value === 'number') {
+      value = value.toString().replace(/\D/g, '');
+      result.push(value);
+    }
```

Comment:
Nesse caso você está substituindo tudo que não for um dígito numérico por ''... de um número? Não entendi a ideia por trás 🤔

Reply:
Não, ele substitui todos os caracteres não númericos por uma string vazia ('').

Reply:
Baaah, verdade. Falha minha 🤦🏻 
Não processei direito a parte dos *não numéricos*, huehue